src/Directory.Build.props

```diff
@@ -48,16 +48,16 @@ Other Changes**
     <AkkaVersion>1.5.8</AkkaVersion>
     <RedisVersion>2.6.111</RedisVersion>
     <DockerVersion>3.125.15</DockerVersion>
-    <NetCoreTestVersion>netcoreapp3.1</NetCoreTestVersion>
+    <NetTestVersion>net7.0</NetTestVersion>
     <NetStandardVersion>netstandard2.0</NetStandardVersion>
-    <NetFrameworkTestVersion>net461</NetFrameworkTestVersion>
+    <NetFrameworkTestVersion>net471</NetFrameworkTestVersion>
```

Comment:
Here also upgraded framework target to match the main project - Not sure if there are any implications.